cmd/kubeadm/app/util/apiclient.go

```diff
@@ -52,3 +53,30 @@ func WaitForAPI(client *clientset.Clientset) {
 		return true, nil
 	})
 }
+
+// WaitForPodsWithLabel will lookup pods with the given label and wait until they are all
+// reporting status as running.
+func WaitForPodsWithLabel(client *clientset.Clientset, name, labelKeyValPair string) {
+	wait.PollInfinite(kubeadmconstants.APICallRetryInterval, func() (bool, error) {
```

Comment:
Can you open an issue so we won't forget? Assign it to me if you like and I should get to it soon.